Tests/GoogleAITests/GenerativeModelTests.swift

```diff
@@ -134,17 +134,61 @@ final class GenerativeModelTests: XCTestCase {
     XCTAssertEqual(promptFeedback.safetyRatings, safetyRatingsNegligible)
   }
 
-  func testGenerateContent_success_unknownEnum() async throws {
+  func testGenerateContent_success_unknownEnum_safetyRatings() async throws {
+    let expectedSafetyRatings = [
+      SafetyRating(category: .harassment, probability: .medium),
+      SafetyRating(category: .dangerousContent, probability: .unknown),
+      SafetyRating(category: .unknown, probability: .high),
+    ]
     MockURLProtocol
       .requestHandler = try httpRequestHandler(
-        forResource: "unary-success-unknown-enum",
+        forResource: "unary-success-unknown-enum-safety-ratings",
         withExtension: "json"
       )
 
-    let content = try await model.generateContent(testPrompt)
+    let response = try await model.generateContent(testPrompt)
+
+    XCTAssertEqual(response.text, "Some text")
+    let candidateSafetyRatings = try XCTUnwrap(response.candidates.first?.safetyRatings)
+    XCTAssertEqual(candidateSafetyRatings, expectedSafetyRatings)
```

Comment:
Could this be combined, or is it too unwieldily? 

```swift
XCTAssertEqual(response.candidates.first?.safetyRatings, expectedSafetyRatings)
```

Below as well

Reply:
I think it's fine to combine them. Done.